### PR TITLE
Hms controller removed

### PR DESCRIPTION
**Pull request by Decoder07**
Fixes:

- Removed HMSController.
- Tested the example app running fine(except the listener functionalities)

### CODE REVIEW

File: example/lib/common/ui/organisms/participant_organism.dart [hunk 1]
```diff
@@ -123,9 +124,9 @@ class _ParticipantOrganismState extends State<ParticipantOrganism> {
 
   void checkButtons() async {
     this.isAudioOn =
-        !(await widget.meetingStore.meetingController.isAudioMute(widget.peer));
+        !(await HmsSdkManager.hmsSdkInteractor!.isAudioMute(widget.peer));
```
Comment (jashan1498) [inline example/lib/common/ui/organisms/participant_organism.dart:127]: add null check above

Comment (Decoder07) [reply example/lib/common/ui/organisms/participant_organism.dart:127]: Ok.

File: example/lib/meeting/meeting_page.dart [hunk 2]
```diff
@@ -440,9 +437,11 @@ class _MeetingPageState extends State<MeetingPage> with WidgetsBindingObserver {
                           Flexible(
                             child: Observer(
                               builder: (_) {
+                                print(
+                                    "Number of people in call ${_meetingStore.peerTracks.length}");
                                 print("rebuilding");
-                                if (!_meetingStore.isMeetingStarted)
-                                  return SizedBox();
+                                // if (!_meetingStore.isMeetingStarted)
```
Comment (jashan1498) [inline example/lib/meeting/meeting_page.dart:443]: remove if not used

File: example/lib/meeting/meeting_store.dart [hunk 3]
```diff
@@ -7,10 +7,11 @@ import 'package:hmssdk_flutter/hmssdk_flutter.dart';
 
 // import 'package:hmssdk_flutter/src/enum/hms_log_level.dart';
```
Comment (jashan1498) [inline example/lib/meeting/meeting_store.dart:8]: clean imports

**Pull request merged**